ui/app/utils/terminal.ts

```diff
@@ -0,0 +1,33 @@
+import { ITerminalOptions, Terminal } from 'xterm';
```

Comment:
Still in the process of reviewing this PR, but could we change the file name of this to `createTerminal`? I'm guessing there's a chance of a namespace collision with this util and the component

Reply:
good idea! i'll change that

Reply:
done in https://github.com/hashicorp/waypoint/pull/2356/commits/e405966f2cc83aaed577e9fdc27b557650cde444 !